templates/frontend/objects/article_details.tpl

```diff
@@ -240,8 +240,26 @@
 											</a>
 										</li>
 									{/foreach}
-							  </ul>
+							  </ul>		
 							</div>
+							{if count($citationDownloads)}
```

Comment:
Can you change this to `{if empty($citationDownloads)}`? It will do the same thing, but if a journal has not enabled the Citation Style Language plugin, the variable `$citationDownloads` won't exist, and `empty` will not throw a warning on an unset variable.

Reply:
done

Reply:
:facepalm: yes, that's right! Thanks for catching that!

Reply:
Thank you @NateWr